crates/next-core/src/next_app/metadata/image.rs

```diff
@@ -53,9 +53,15 @@ pub async fn dynamic_image_metadata_source(
 
     let use_numeric_sizes = ty == "twitter" || ty == "openGraph";
     let sizes = if use_numeric_sizes {
-        "data.width = size.width; data.height = size.height;"
+        "data.width = size.width; data.height = size.height;".to_string()
     } else {
-        "data.sizes = size.width + \"x\" + size.height;"
+        let sizes = if ext == "svg" {
```

Comment:
As far as I can tell this case can never happen... but in order to make it consistent with the webpack loader applied it here as well.